templates/financial-services/index.html

```diff
@@ -532,37 +581,39 @@ <h2>Intelligent finance</h2>
         <a href="/kubeflow">machine learning processes with Kubeflow</a>. With  Ubuntu, you benefit from perfect multi-cloud portability of AI/ML workloads.</p>
     </div>
     <div class="col-4 u-align--center u-hide--small">
-      <ul class="p-inline-images u-no-margin">
-        <li class="p-inline-images__item">
+      <div class="p-logo-section u-no-margin">
```

Comment:
Could we make the logos be centrally aligned or make two logos in one line? What do you think?

Reply:
For sure! I have made these changes now